gno.land/pkg/integration/testdata/prevrealm.txtar

```diff
@@ -108,10 +115,14 @@ package foo
 import "gno.land/r/myrlm"
 
 func A() string {
+	crossing()
+
 	return myrlm.A()
```

Comment:
did this work? hmm... should have failed because myrlm.A is crossing so it should have to be called with cross(myrlm.A)().